x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/constants.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const TOTAL_DOCUMENTS = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.analytics.totalDocuments',
+  { defaultMessage: 'Total Documents' }
+);
+
+export const TOTAL_API_OPERATIONS = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.analytics.totalApiOperations',
+  { defaultMessage: 'Total API Operations' }
+);
+
+export const TOTAL_QUERIES = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.analytics.totalQueries',
+  { defaultMessage: 'Total Queries' }
+);
+
+export const TOTAL_CLICKS = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.analytics.totalClicks',
+  { defaultMessage: 'Total Clicks' }
+);
```

Comment:
These constants are also used in the Analytics view so I figured I'd do myself a favor and start a constants file early.